.github/workflows/build_executables.yml

```diff
@@ -31,10 +31,10 @@ jobs:
 
       - name: 🔨 Build Apk
         shell: bash
-        run: flutter build apk --debug
+        run: flutter build apk --profile
```

Comment:
I'm surprised. Does not need release signatures?

Reply:
We only configured signing for `release`, `assembleProfile` doesn't seem to use the signing config:

https://github.com/nt4f04uNd/sweyer/blob/b96a5ee3eead62e546059a13b4071c59f7a11454/android/app/build.gradle#L60-L61